supertokens_python/recipe/thirdpartyemailpassword/api/thirdparty_api_implementation.py

```diff
@@ -31,10 +27,7 @@ def get_interface_impl(
     implementation.disable_apple_redirect_handler_post = api_implementation.disable_apple_redirect_handler_post
 
     if not implementation.disable_sign_in_up_post:
```

Comment:
You do not need this `if` statement. It's there here cause earlier we would create a wrapper function if it was not disabled. But now we don't need to make a wrapper function, so we can remove this and follow the same pattern as we are for the other api functions

Reply:
Got it.